### PR TITLE
Make User and Group kinds not default allowed by rules

### PR DESCRIPTION
**Pull request by freben**
This retroactively changes two changelogs and has changesets for that ... not sure if that's just silly, let me know what you think :)

### CODE REVIEW

File: .changeset/clean-lions-taste.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      rules:
+        allow: [User, Group]
+```
+
+This rule change does NOT affect entity providers, only things that are emitted
+by entity processors.
+
+We recommend that this change is applied to your own Backstage repository, since
+it makes it impossible for regular end users to affect your org data through
+e.g. YAML files. Do to so, remove the two kinds from the default rules in your config:
```
Comment (Rugvip) [inline .changeset/clean-lions-taste.md:34]:
```suggestion
e.g. YAML files. To do so, remove the two kinds from the default rules in your config:
```

**Review by Rugvip — APPROVED** [review]
Yaaaaaas

File: .changeset/clean-lions-taste.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+And for any location that in any way results in org data being ingested, add the corresponding rule to it:
+
+```diff
+ catalog:
+   locations:
+     - type: github-org
+       target: https://github.com/my-org-name
++      rules:
++        allow: [User, Group]
```
Comment (julioz) [inline .changeset/clean-lions-taste.md:51]:
```suggestion
+        - allow: [User, Group]
```
I think we might have missed the `-` for the rules list. There are other spots in this PR that also missed the character

Comment (freben) [reply .changeset/clean-lions-taste.md:51]: Oh man the copy-paste is real here. Thanks for notifying me, especially before release today!

**Pull request merged**